### ISSUE TITLE
Updated embed method from iframe to JS

### ISSUE THREAD

**Issue by barnabyalter** [NYULibraries/primo-explore-libraryh3lp-widget#10]
Libraryh3lp has a never pure javascript recommended way of implementing their code, which as I gather allows the admins of the system to route the widget to the correct queue.

We've already done this on library.nyu.edu:

https://github.com/NYULibraries/library.nyu.edu/commit/ec252ed4d8a048451c60b1d8497b7b215f47c9e5

**Comment by barnabyalter** [comment]
@etgrieco I keep forgetting about this. Could you make this update in this module and deploy it across NYU/NYUAD/NYUSH/NYSID?

Noting that NYSID uses different queue/id/view vars

Thanks!

**Comment by etgrieco** [comment]
Yes! Definitely will get this done -- something I can jump in-and-out of today
